src/components/DiagnosticsButtonRun.vue

```diff
@@ -1,11 +1,19 @@
 <script lang="ts">
+import { ipcRenderer } from 'electron';
 import Vue from 'vue';
-export default Vue.extend({ name: 'diagnostics-button-run' });
+export default Vue.extend({
+  name:    'diagnostics-button-run',
+  methods: {
+    onClick() {
+      ipcRenderer.send('diagnostics/run');
+    },
```

Comment:
Would it make sense to communicate via the command server api instead of ipc?

Reply:
Yes; I'm not sure I did the dumb thing here. I don't need to thread through the main process here…

Reply:
Fixed in da002f7c1f0fb21191d9e513689e674546cd5074.